### DISCUSSION TITLE
Secureboot build - automation and logging

### DISCUSSION THREAD

**Discussion by NeilHanlon** [rocky-linux/peridot#136] [Features]
We need the ability to automatically build, test, and release updates to secureboot-affected pacakges: kernel, fwupd, grub2, shim. (UKIs in the future).

Need to understand requirements not only from  RESF policy wise but UEFI committee, too.

**Comment by SherifNagy** [comment]
Also we need to keep the following:
*A DB that contains all the signed efi binary hashes
*A sigstore for what is being signed
